docs/source/explore-notebooks-and-manage-dependencies.md

```diff
@@ -1,22 +1,58 @@
+
+2. [training.ipynb](../../notebooks/training.ipynb) for training the model for MNIST Classification and storing it locally or in cloud object storage
+
+
+# Manage dependencies
+
+Reproducibility and shareability of notebooks is very important if you want to allow others to repeat your experiments and avoid issues due to dependencies management.
+Dependencies of jupyter notebook are managed using the extension for JupyterLab called [jupyterlab-requirements][1].
+
+This extension can be used in different ways:
+
+- from notebook cells using `%horus` magic commands ([check video](https://www.youtube.com/watch?v=FjVxNTXO70I)).
+
+- from UI using the button that appears when the notebook is opened.
+
+For example if you click on the `Manage Dependencies` button to interact with the JupyterLab extension for dependency management. \\
 
 <div style="text-align:center">
-<img alt="Jupyter Hub UI" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/JupyterLabRequirementsExtension.jpg">
+<img alt="JupyterLab Requirements UI" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/JupyterLabRequirementsExtension.jpg">
 </div>
 
-You can use this extension for each notebook to guarantee all notebooks have the correct dependencies and kernel.
+- from CLI directly from terminal or integrated in pipelines ([check video](https://www.youtube.com/watch?v=fW0YKugL26g&t)).
 
-The notebooks that will be used in an AI Pipeline are:
+You can use this extension for each of your notebook to guarantee they have the correct dependencies and kernel.
 
-1. [download_dataset.ipynb](../../notebooks/download_dataset.ipynb) for downloading the MNIST dataset from Tensorflow. _NOTE: in [Elyra][2] this step can also be performed with the provided [Python script](../../data/download_dataset_from_tf.py)_
 
-2. [training.ipynb](../../notebooks/training.ipynb) for training the model for MNIST Classification and storing it locally or in cloud object storage
+# Run the notebooks
+
+For the notebooks in this tutorial, the dependencies have been already set.
+You can check the status of your notebook running `%horus check`:
+
+<div style="text-align:center">
+<img alt="Horus check command" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/JupyterLabHorusCheck.jpg">
+</div>
+
+You can check the dependencies content of your notebook running `%horus show`:
+
+<div style="text-align:center">
+<img alt="Horus show command" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/JupyterLabHorusShow.jpg">
+</div>
+
+If you want to run the notebooks, the only thing you have to do is to set the kernel running `%horus set-kernel` in one cell of the notebook.
```

Comment:
not sure, because we don't want that to be run in the AI pipeline. I think Jupiter magic commands are not run through papermill but I need to test that.